pyEpiabm/pyEpiabm/tests/test_sweep/test_host_progression_sweep.py

```diff
@@ -13,34 +14,34 @@ def setUp(self) -> None:
         """Sets up a population we can use throughout the test.
         2 people are located in one microcell.
```

Comment:
```suggestion
        3 people are located in one microcell.
```
Docstring doesn't match test

Reply:
In fact you should describe that you are making two separate populations